docs/source/basic_usage.rst

```diff
@@ -0,0 +1,106 @@
+Basic Usage
+===========
+
+``pygeoprocessing`` uses GDAL to read and write all GDAL-supported file
+formats. The utility functions
+:func:`get_raster_info <pygeoprocessing.get_raster_info>` and
+:func:`get_vector_info <pygeoprocessing.get_vector_info>` read a file’s
+metadata into a dictionary. These values can be used as parameters for a
+variety of ``pygeoprocessing`` functions where properties like pixel sizes,
+bounding boxes, and nodata values need to be defined.
+
+.. code::
+
+    import pygeoprocessing
+
+.. code::
+
+    raster_a_path = 'dem_utm.tif'
+    raster_b_path = 'mswep.tif'
+    raster_info = pygeoprocessing.get_raster_info(raster_a_path)
+    raster_info
```

Comment:
Suggest adding `>>> ` to the start of each line to resemble the python REPL and make the distiction between the code and the output below